app/models/office.rb

```diff
@@ -61,8 +59,12 @@ def field_order
     ]
   end
 
+  def telephone_number=(new_phone_number)
+    super cleanup_telephone_number(new_phone_number)
+  end
+
   def telephone_number
-    super.try { |x| x.gsub(' ', '') }
+    return format_telephone_number(cleanup_telephone_number(super))
```

Comment:
Yes as we discussed but I changed my mind. I felt it's cheap enough that we might as well put it in for now. This way we don't *have* to do the upfront clean-up and so there's no need to answer the questions

>  is it a rake task? Is it a migration?  Is it a one-off script? If so where do I put it?
> Can it just be a one-liner we run in the console? If so where do I record this so it doesn't get forgotten?

Right now all we have to do is do a standard deploy including re-indexing to ElasticSearch and job done.